geps/gep-1619.md

```diff
@@ -74,28 +74,122 @@ While session persistence uses attributes in the application layer, session affi
-| eBPF 	| Cilium 	| No 	| Source IP Address 	| Cilium does not have a true session persistence feature, but does implement [session affinity by default](https://docs.cilium.io/en/v1.12/gettingstarted/kubeproxy-free/#id2) using source IP address. 	|
-| Google CloudRun 	| Google Kubernetes Engine 	| [Cookie-Based](https://cloud.google.com/run/docs/configuring/session-affinity) 	| No 	| Google Cloud Run uses the terminology "session affinity" for what we define as session persistence. 	|
-| Pipy 	| Flomesh Service Mesh 	| ? 	| ? 	| ? 	|
-| HAProxy 	| HAProxy Ingress 	| [Cookie-Based](https://docs.haproxy.org/2.6/configuration.html#4-option%20forwardfor) 	| [Stick Tables](https://cbonte.github.io/haproxy-dconv/2.4/configuration.html#4-stick-table) using connection attributes 	| HAProxy implements cookie-based stateful sessions via the [forwardfor option](https://docs.haproxy.org/2.6/configuration.html#4-option%20forwardfor). 	|
-| Kong 	| Kong 	| Cookie-Based 	| ? 	| Kong has the [session plugin](https://docs.konghq.com/hub/kong-inc/session/) which manages persistent sessions with cookie-based stateful sessions. 	|
-| Traefik Proxy 	| Traefik 	| [Cookie-Based](https://doc.traefik.io/traefik/routing/services/#sticky-sessions) 	| No 	| [Docs](https://traefik.io/glossary/what-are-sticky-sessions/) mention consistent hashing, but Traefik [only offers round robin](https://doc.traefik.io/traefik/routing/services/#load-balancing) (not consistent). 	|
-| Java Servlet 	| N/A 	| [Cookie-Based](https://docs.oracle.com/javaee/7/api/javax/servlet/http/HttpServletRequest.html#getSession--), [URL Encoded](https://docs.oracle.com/javaee/7/api/javax/servlet/http/HttpServletRequest.html#isRequestedSessionIdFromURL--) 	| N/A 	| Java Servlets don't natively support proxy functions. 	|
-| Apache httpd 	| N/A 	| [Cookie-Based](https://httpd.apache.org/docs/2.4/mod/mod_proxy_balancer.html), [URL Encoded](https://httpd.apache.org/docs/2.4/mod/mod_proxy_balancer.html) 	| N/A 	|  N/A	|
-
+In this section, we will describe how implementations achieve session persistence and affinity, along with a breakdown of related configuration options. Input from implementations is appreciated to complete this information.
+
+In the following tables, we will example two types of APIs:
+1. Dataplane APIs
+2. Implementation APIs
+
+Generally, the implementation API programs the dataplane API; however these two are not always clearly separated. The two types of APIs can use different API structures for configuring the same feature. Examining the dataplane APIs helps to remove the layer of API abstraction that implementations provide. Removing this layer avoids situations where implementations don’t fully implement all capabilities of a dataplane API or obfuscate certain configuration around session persistence. On the other hand, examining implementation APIs provides valuable data points in what implementations are interested in configuring.
+
+**Session Persistence**
```

Comment:
Review the tables here: https://github.com/gcs278/gateway-api/blob/gep-1619-cookie-configuration/geps/gep-1619.md